### ISSUE TITLE
Tandy and Ample Lite

### ISSUE THREAD

**Issue by TandyColorComputer3** [ksherlock/ample#27]
Thank you ksherlock, Ample Lite has been easy to modify so far. 

I'm editing your files to support a Tandy-centric version of Ample Lite  - to include something like the below.  I love the concept of making MAME more accessible with menus - but also showing the command line instructions. Looking forward to future developments.

TRS-80
 - Model I Level I
 - Model I Level II
 - Model 200
 - Model II
 - Model III

Tandy Color Computer
 - CoCo 1
 - MC-10
 - CoCo 2
 - CoCo 2 w/6309
 - Deluxe Color Computer 
 - CoCo 3
 - CoCo 3 w/6309

Dragon
 - Dragon32
 - Dragon64
 - Dragon200
 - Tano Dragon
 - Dragon Professional

Tandy 1000 Series
 - Tandy 1000 HX
 - Tandy 1000 RL
 - Tandy 1000 SL/2
 - Tandy 1000 TL/2
 - Tandy 1000 SX
 
Tandy Other
 - Tandy 2000
 - WP-2
 - WP-3

![Screenshot 2024-03-06 at 7 39 54 PM](https://github.com/ksherlock/ample/assets/64712606/cf2f57ed-95b9-4c46-ad24-84a0e6b78461)

**Comment by ksherlock** [comment]
I didn't realize MAME emulated the coco and friends.  I'll include them in the next version.

**Comment by TandyColorComputer3** [comment]
Oh cool, great to hear.

A lot of the casual coco users avoid mame due to the setup. The primary ease-of-use dedicated emulators have their own issues - VCC scales pixels improperly and is windows only. XRoar is quirky due to its dragon lineage and is still fine-tuning CoCo3 and 6309 support. The bookmarks feature will be handy to save the different HW setups involved in supporting alternate ROMs, hard drive images, and NitrOS-9 operating system.

Tim Lindner is the technical brains behind a lot of coco mame development should you have questions that require that level of expertise.

https://github.com/tlindner
http://tlindner.macmess.org/

**Comment by ajacocks** [comment]
Yeah, definitely agreed. Ample really exposes the incredible work that the MAME devs do to a wider audience.

**Comment by ajacocks** [comment]
Interesting. Any idea of why it's not being recognized?

**Comment by TandyColorComputer3** [comment]
I'm scanning the coco .plist, not sure yet.  The DISTO Super Controller would be a workaround, but it also doesn't show floppy with the multipak. Glenside IDE is not showing a hard disk either.  So the multipak is not set up.

**Comment by ksherlock** [comment]
It looks like the floppy disks embedded within 6 levels of slots (eg, -ext:multi:slot4:fdc:wd17xx:1) and when I'm building the property list it doesn't go deep enough to find them.

**Comment by ksherlock** [comment]
I did a stealth update on the release to include the coco floppies

**Comment by TandyColorComputer3** [comment]
Thank you for the quick fix - mpi launching disks/HDs and working great.  For future Tandy releases, I recommend:

- **coco3h** (this is 6309 processor version that is the new modern standard, a Hitachi-made m6809 clone that runs more efficiently - and as it turned out, has a Native mode with hidden instructions. Required for the fastest performance with NitrOS9 and a [AAA game](https://nickmarentes.com/ProjectArchive/gunstar.html))
- **mc10** (formerly the butt of doorstop jokes, the 6803-based MC-10 Micro Color Computer has become a favorite of the engineer class who have used it to test out concepts before bringing those ideas to the CoCo)
- **coco3dw1**   (I think this presets drivewire w/hdb-dos, DW is the protocol to communicate with a host pc / internet)
- **coco3hdb** (I think hdb-dos w/o dw. some folks use the [additional features](https://tinyurl.com/2fcmpejd))

**Comment by ksherlock** [comment]
coco3h and mc10 were added in the Ample 0.265.  The dw1/hdb machines were removed from mame a few months ago, in favor of selecting the disk drive ROM.  Ample 0.265 now has a UI to select the disk drive ROM.

<img width="576" alt="image" src="https://github.com/ksherlock/ample/assets/964904/32861f6e-9871-46f4-94c3-fe76e57b300d">

**Comment by ajacocks** [comment]
The interface looks good to me. Clear.
